tests/WE2E/monitor_jobs.yaml

```diff
@@ -0,0 +1,54 @@
+# This is an example yaml file showing the various entries that can be created for tracking jobs by monitor_jobs.py
```

Comment:
How should this file be updated when changes are made to monitor_jobs.py such that this file is modified?

Can that be a GitHub action?

Reply:
It is unclear to me how that would be handled in an automatic way. Is it worth the effort for something that will likely not change very often, if at all?

Reply:
It could be nice just to drop in a comment on how this appeared here so that future you is not completely baffled. ;)